pages/api/xAPI/statements/index.ts

```diff
@@ -59,27 +61,20 @@ async function runMiddleware(
 async function handleGET(req: NextRequest, res: NextResponse): void {
   try {
     // await runMiddleware(req, res, sanitizeQueryParams);
-    // await runMiddleware(req, res, validateQueryParams);
     // await runMiddleware(req, res, cors);
-    await dbClient.connect();
+    helpers.validateQueryParams(req, res, (err) => {if (err) throw err});
     const queryOptions = generateQueryParams(req.query);
     const rows = await dbClient
-      .db("xAPI")
-      .collection("statements")
-      .find(queryOptions.params)
-      .sort(queryOptions.sort)
-      .toArray();
-    rows.forEach((row) => {
-      delete row["_id"]; //_id is a mongoDB parameter outside the scope of xAPI
-    });
+      .statement
+      .findMany();
+    const statements = rows.map((row) => row.statement);
     res.status(200).json({
-      statements: rows,
+      statements: statements,
```

Comment:
Interestingly, in javascript/typescript world:

```
res.status(200).json({
      statements: statements,
```

and 

```
res.status(200).json({
      statements,
```

are equivalent.